File: Data Structures & Algorithms/concatenation-of-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    getConcatenation(nums) {
        let len = nums.length;
        for(let i = 0; i < len; i++){
            nums.push(nums[i]);
        }
        return nums;
    }
}

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.cs
public class Solution {
    public int FindKthLargest(int[] nums, int k) {
        var maxHeap = new PriorityQueue<int,int>(Comparer<int>.Create((x,y) => y.CompareTo(x)));

        foreach (int num in nums){
            maxHeap.Enqueue(num, num);
        }

        while(k > 1){
            maxHeap.Dequeue();
            k--;
        }

        return maxHeap.Peek();
    }
}
